src/kibana/plugins/dashboard/components/modal/modal.js

```diff
@@ -0,0 +1,122 @@
+define(function (require) {
+   
+    var app = require('modules').get('app/dashboard');
+    require('plugins/dashboard/components/modal/validators');
+    require('plugins/dashboard/components/modal/kbnIndexService');
+    require('plugins/dashboard/components/modal/ElasticSearchFields');
```

Comment:
So ```src/kibana/plugins/dashboard/components/modal/ElasticSearchFields.js```.  Since these are netmon specific wouldn't it make sense to put them in ```/netmon_libs/dashboard/components/modal/ElasticSearchFields.js``` or similar